app/templates/main/map.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+<script src="http://maps.googleapis.com/maps/api/js"></script>
+<script>
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+function initialize() {
+var initial_coords = new google.maps.LatLng(39.952, -75.195);
+var mapProp = {
+center: initial_coords,
+zoom:17,
+mapTypeId: google.maps.MapTypeId.ROADMAP
+};
+map=new google.maps.Map(document.getElementById("googleMap"),mapProp);
+}
+function reinitialize() {
+var initial_coords = new google.maps.LatLng(39.952, -75.195);
+var mapProp = {
+center: initial_coords,
+zoom:17,
+mapTypeId: google.maps.MapTypeId.ROADMAP
+};
+var map=new google.maps.Map(document.getElementById("googleMap"),mapProp);
+if (geocoder) {
+console.log(address);
+geocoder.geocode( { 'address': address}, function(results, status) {
+if (status == google.maps.GeocoderStatus.OK) {
+if (status != google.maps.GeocoderStatus.ZERO_RESULTS) {
+map.setCenter(results[0].geometry.location);
+}
+else {
+alert("Address not found!");
```

Comment:
Let's try to use flashing and not alerts. You can see the html you need to output to trigger a flash in app/templates/partials/_flashes.html. It would be great if you could define a JavaScript function which outputs that exact html (since Max already styled it). To see an example of what I mean, check out https://github.com/hack4impact/2015-project-tester/blob/master/app/templates/submission.html#L46

To see this in practice, go to http://project-testing.herokuapp.com/ and give it a URL. Notice how error messages are "flashed" to the top of the webpage. When you first load the page, you see the message "Messages will show up here. Built by Ben Sandler."